prepare-iso.sh

```diff
@@ -188,32 +212,22 @@ done
 # See if we can find an eligible installer.
 # If successful, then create the iso file from the installer.
 
-if installerExists "Install macOS Big Sur.app" ; then
+if installerExists "Install macOS Monterey.app" ; then
+  createISO "Install macOS Monterey.app" "Monterey"
+elif installerExists "Install macOS Big Sur.app" ; then
```

Comment:
I replaced all the nested `else if` statements with `elif`, which I find to be much more readable and easier to maintain. I'm happy to yank this change out of the pull request if that would be preferred.